src/lib/wdi5-control.ts

```diff
@@ -659,6 +659,23 @@ export class WDI5Control {
         }
 
         const _result = (await clientSide_getControl(controlSelector, this._browserInstance)) as clientSide_ui5Response
+
+        // When chromedriver service is not used the domElement is not set accordingly via devtool protocol
+        // Therefore we get element reference by calling browser execute function manually
+        if (_result.status === 0 && !_result.domElement[ELEMENT_KEY]) {
+            const elementReference = (await this._browserInstance.execute((id) => {
+                const webElement: Node = document.evaluate(
```

Comment:
black magic here :)
I would tend to see this async, meaning `await this._browserInstance.executeAsync` but I'm ok with this in here b/c this part is executing client-side already is in a "needs to run" in order thing

Reply:
i take this back - for some reason, `await`ing`this._browserInstance.execute` is necessary - guessing an issue with the event loop tick.
anyway, resolving this.